GameData/RealismOverhaul/Localization/en-us-Engines.cfg

```diff
@@ -32,10 +32,10 @@ Localization
 		#roAestusDesc = A hypergolic upper stage engine. The baseline Aestus engine is pressure fed and is used on the Ariane 5 G, GS and ES series, while the Aestus II (also known as RS-72) variant is pump-fed with better overall performance.
 		//		Agena
 		#roAgenaTitle = XLR81 (Agena) Vacuum Engine
-		#roAgenaDesc = A gas-generator nitric acid/UDMH vacuum engine used on Agena. The XLR81 family was derived from the Bell Hustler Rocket Engine, originally developed for use on an air-to-surface missile. Early engines were nearly identical to the Hustler engine, while later variants offered new capabilities and improved performance. Engine restart was introduced on the Agena B's XLR81-BA-7 (Model 8081). The XLR81-BA-11 (Model 8096) used on Agena D used propellant sumps to eliminate the need for ullage thrust. The XLR81-BA-13 (Model 8247) powered the Gemini Agena Target Vehicle (a modified Agena D) and was rated for up to 14 restarts.
+		#roAgenaDesc = A missile-derived gas-generator engine used on Agena. Early engines were nearly identical to the military version, while later variants improved capabilities and performance. Engine restart was added on the Agena B's XLR81-BA-7 (Model 8081). The XLR81-BA-11 (Model 8096) used on Agena D had propellant sumps to handle ullage. The XLR81-BA-13 (Model 8247) powered the Gemini Agena Target Vehicle (a modified Agena D) and was rated for up to 14 restarts.
```

Comment:
I think this changes the meaning too strongly? "Missile-derived" make people think of, well, ballistic missiles, rather than AGMs. 
Instead try "Originally developed to power a standoff weapon for the B-58 Hustler, the XLR81 is a gas-generator engine with storable propellants. Later space variants improved performance and capability."
then it's fine except I'd go back from 'had propellant' to 'used propellant'.